### PR TITLE
Remove --webdriver-binary

### PR DESCRIPTION
**Pull request by jeffcarp**
Via #testing, we were passing the wrong value to
--webdriver-binary for Geckodriver, causing webdriver tests
in Firefox to fail. Now that `wpt run` supports downloading
a webdriver binary itself, we can remove this option.

cc @AutomatedTester

### CODE REVIEW

File: run/running.example.ini [hunk 1]
```diff
@@ -1,11 +1,9 @@
 [default]
 build_path = $HOME/wptdbuild
-wpt_path = $HOME/gh/w3c/web-platform-tests
-wptd_path = $HOME/gh/w3c/wptdashboard
+wpt_path = $HOME/web-platform-tests
+wptd_path = $HOME/wptdashboard
```
Comment (jeffcarp) [inline run/running.example.ini]: This change is unrelated. I always have to change this when setting up a bot. Sneaking in a change to the default here.

**Comment by AutomatedTester** [comment]
We could leave code to handle specific webdriver binaries if people wanted to point to things specifically but that can be done in a follow up.

**Comment by bobholt** [comment]
Agree with @AutomatedTester, but this is good. :+1:
